pkg/controller/utils/delayingreconciler.go

```diff
@@ -0,0 +1,39 @@
+package utils
+
+import (
+	"context"
+	log "github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"time"
+)
+
+const requeueDelay = time.Minute * 2
+
+type delayingReconciler struct {
+	inner  reconcile.Reconciler
+	logger log.FieldLogger
+}
+
+func NewDelayingReconciler(r reconcile.Reconciler, logger log.FieldLogger) reconcile.Reconciler {
+	return delayingReconciler{
+		inner:  r,
+		logger: logger.WithField("DelayingReconciler", requeueDelay),
+	}
+}
+
+func (d delayingReconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
+	result, err := d.inner.Reconcile(ctx, request)
+
+	if errors.IsForbidden(err) {
+		// HIVE-1895:
+		// a) Do not consider this an error. Client credentials are bad.
+		// b) Requeue and pray credentials are better next time
+		d.logger.Info("Requeueing request due to bad permissions")
+		return reconcile.Result{RequeueAfter: requeueDelay}, nil
```

Comment:
Note from digging: we can't return an error here because returning an error means the request would be enqueued w/ rate limiting per https://github.com/kubernetes-sigs/controller-runtime/blob/f0351217e9e026533aece74d054b23cae5441659/pkg/internal/controller/controller.go#L318-L344 which would ignore our `requeueAfter`.